app/controllers/xlsx_controller.rb

```diff
@@ -0,0 +1,41 @@
+class XlsxController < ApplicationController
+  def tojson
+
+    excel = Roo::Excelx.new('/home/felix/se/PhD-Application-Review-System/Dummy_data.xlsx')
```

Comment:
This needs to be changed to a relative path. For now you can probably try relative refrencing instead. Line `../../Dummy_data.xlsx`